source/patterns/@aws-solutions-constructs/aws-events-rule-kinesisfirehose-s3/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@aws-solutions-constructs/aws-events-rule-kinesisfirehose-s3",
+  "version": "1.67.0",
+  "description": "CDK Constructs for Amazon CloudWatch Events Rule to Amazon Kinesis Firehose to Amazon S3 integration.",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/awslabs/aws-solutions-constructs.git",
+    "directory": "source/patterns/@aws-solutions-constructs/aws-events-rule-kinesisfirehose-s3"
+  },
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
+  },
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "tsc -b .",
+    "lint": "eslint -c ../eslintrc.yml --ext=.js,.ts . && tslint --project .",
+    "lint-fix": "eslint -c ../eslintrc.yml --ext=.js,.ts --fix .",
+    "test": "jest --coverage",
+    "clean": "tsc -b --clean",
+    "watch": "tsc -b -w",
+    "integ": "cdk-integ",
+    "integ-assert": "cdk-integ-assert",
+    "integ-no-clean": "cdk-integ --no-clean",
+    "jsii": "jsii",
+    "jsii-pacmak": "jsii-pacmak",
+    "build+lint+test": "npm run jsii && npm run lint && npm test && npm run integ-assert",
+    "snapshot-update": "npm run jsii && npm test -- -u && npm run integ-assert"
+  },
+  "jsii": {
+    "outdir": "dist",
+    "targets": {
+      "java": {
+        "package": "software.amazon.awsconstructs.services.eventsrulekinesisfirehoses3",
+        "maven": {
+          "groupId": "software.amazon.awsconstructs",
+          "artifactId": "eventsrulekinesisfirehoses3"
+        }
+      },
+      "dotnet": {
+        "namespace": "Amazon.Constructs.AWS.eventsrulekinesisfirehoses3",
+        "packageId": "Amazon.Constructs.AWS.eventsrulekinesisfirehoses3",
+        "signAssembly": true,
+        "iconUrl": "https://raw.githubusercontent.com/aws/aws-cdk/master/logo/default-256-dark.png"
+      },
+      "python": {
+        "distName": "aws-solutions-constructs.aws-events-rule-kinesisfirehose-s3",
+        "module": "aws_solutions_constructs.aws_events_rule_kinesisfirehose_s3"
```

Comment:
Use `kinesis-firehose` instead of `kinesisfirehose`

Reply:
Done